docs/CAVEATS.md

```diff
@@ -0,0 +1,17 @@
+# Caveats
+## Only iceberg V2 table supported
+This connector only writes using iceberg table V2 spec (no rewrite for delete and upsert - merge on read only)
```

Comment:
what you mean by this one? updates are supported with this setting https://github.com/memiiso/debezium-server-iceberg/blob/master/docs/DOCS.md#upsert

maybe it needs rewording.

Reply:
this blog post might answer some of your questions, maybe give idea about different configurations 
https://debezium.io/blog/2021/10/20/using-debezium-create-data-lake-with-apache-iceberg/

Reply:
i think i got what you mean. its using merge on read for deletes. deletes are added as quality delete(delta file)

Reply:
Yes, maybe I will take care of the wording to explain more